apps/omg_watcher/lib/omg_watcher/ethereum_event_aggregator.ex

```diff
@@ -84,6 +84,10 @@ defmodule OMG.Watcher.EthereumEventAggregator do
 
   def init(opts) do
     contracts = opts |> Keyword.fetch!(:contracts) |> Map.values() |> Enum.map(&from_hex(&1))
+    # This is a pure hack....currently exit game would be double configed by
+    # Config.contracts().payment_exit_game and exit_games
+    contracts = Enum.uniq(contracts)
```

Comment:
I don't understand why is exit games addresses configured in multiple places (?) 
Can we unify this so we can obtain exit game address for a given tx_type from single, meaningfull module?